ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -85,9 +87,11 @@
   // the total number of data chunks retrieved so far (and may or may not have been written out yet).
   private int numChunksRetrieved;
   // the maximum size of a data chunk in bytes
-  private int chunkSize = CompositeBlobInfo.UNDEFINED_CHUNK_SIZE;
+  private long chunkSize;
```

Comment:
Alright - anything that works, not really particular - was just curious why this was changed.